chart/Chart.yaml

```diff
@@ -22,9 +22,8 @@ version: 1.0.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.0.0"
+appVersion: "2.0.0-develop"
```

Comment:
will have to think about also keeping older Helm chart releases

I'd need to take a look at how it's done for OpenEBS

Reply:
Yep, would be good to have a plan. Also would we want to include this in the OpenEBS charts?

Reply:
I'll put this up for discussion in the community meeting, if the answer is yes, it'd become another sub Helm chart under OpenEBS chart.